distribution/pom.xml

```diff
@@ -125,24 +117,71 @@
             <groupId>org.apache.iotdb</groupId>
             <artifactId>library-udf</artifactId>
             <version>${project.version}</version>
-            <type>jar</type>
         </dependency>
         <dependency>
             <groupId>org.apache.iotdb</groupId>
             <artifactId>spark-iotdb-connector_2.11</artifactId>
             <version>${project.version}</version>
-            <type>jar</type>
         </dependency>
         <dependency>
             <groupId>org.apache.iotdb</groupId>
             <artifactId>spark-iotdb-connector_2.12</artifactId>
             <version>${project.version}</version>
-            <type>jar</type>
```

Comment:
Why do you remove the type?

Reply:
Usually you don't specify it, if it's "jar" as that is the default.